---
exp_x_distance: 1.5
x_max_distance: 1.8
x_min_distance: 1.2
y_max_distance: 0.3
max_angle: 0.3
max_step: 200
model_params: "src/rl_enviroment/config/real_world.yaml"
leader_sub_topic: "/agv2/odom"
follower_sub_topic: "/agv1/odom"
leader_pub_topic: "/agv2/cmd_vel"
follower_pub_topic: "/agv1/cmd_vel"
leader_link: "/agv2/base_link"
follower_link: "/agv1/base_link"
low_bound:
  - -0.3
  - -0.4
  - -0.15
robot2_low_bound:
  - -0.2
  - -0.2
  - -0.1
high_bound:
  - 0.3
  - 0.4
  - 0.15
robot2_high_bound:
  - 0.2
  - 0.2
  - 0.1
...